client/ayon_maya/plugins/publish/extract_pointcache.py

```diff
@@ -515,8 +519,9 @@ def get_attribute_defs(cls):
 
 
 class ExtractAnimation(ExtractAlembic):
-    label = "Extract Animation (Alembic)"
+    label = "Extract Animation"
```

Comment:
```suggestion
    label = "Extract Animation (Alembic)"
```
Without this suffix the user would just be presented with a checkbox `Extract Animation` which admittedly is unclear... which of the extractions? Alembic, FBX, USD?